### PR TITLE
New endpoint f5d

### PR DESCRIPTION
**Pull request by JoanaFigueira**

### CODE REVIEW

File: infoenergia_api/contrib/f5d.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        filters = [
+                ('active', '=', True),
+                ('state', '=', 'activa'),
+                ('empowering_profile_id', '=', 1),
+                ('cups', '=', self.name)
+            ]
+
+        contract = contract_obj.search(filters)
+        if contract:
```
Comment (juan-pe) [inline infoenergia_api/contrib/f5d.py:69]: return bool(contract)

**Review by juan-pe — CHANGES_REQUESTED** [review]
only is_valid_empowering change :smiley: 

super code!

File: infoenergia_api/contrib/f5d.py (hunk 1 above)
Comment (JoanaFigueira) [reply infoenergia_api/contrib/f5d.py]: done :)

**Review by juan-pe — APPROVED** [review]
nice!! :smile:

**Pull request merged**